whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/Inet6numSpec.groovy

```diff
@@ -1564,13 +1564,15 @@ class Inet6numSpec extends BaseQueryUpdateSpec {
         ack.summary.assertSuccess(0, 0, 0, 0, 0)
         ack.summary.assertErrors(2, 0, 2, 0)
 
-        ack.countErrorWarnInfo(2, 0, 0)
+        ack.countErrorWarnInfo(4, 0, 0)
         ack.errors.any { it.operation == "Modify" && it.key == "[inet6num] 2001:600::/32" }
         ack.errorMessagesFor("Modify", "[inet6num] 2001:600::/32") ==
-                ["status value cannot be changed, you must delete and re-create the object"]
+                ["status value cannot be changed, you must delete and re-create the object",
+                "\"mnt-lower:\" attribute not allowed for resources with \"ASSIGNED:\" status"]
         ack.errors.any { it.operation == "Modify" && it.key == "[inet6num] 2001:600::/30" }
         ack.errorMessagesFor("Modify", "[inet6num] 2001:600::/30") ==
-                ["status value cannot be changed, you must delete and re-create the object"]
+                ["status value cannot be changed, you must delete and re-create the object",
+                "\"mnt-lower:\" attribute not allowed for resources with \"ASSIGNED:\" status"]
```

Comment:
Update test data (i.e. remove "mnt-lower") rather than having this extra (not relevant) error in the test

Reply:
Remove "mnt-lower"

Reply:
Done